js_functions/task3.js

```diff
@@ -0,0 +1,9 @@
+function printNum (num) {
```

Comment:
Just looking at name `printNum` I would expect it prints one number. But you function behaves in different name. Pick better name like countDown etc